--- warzone2100-2.1_rc1/icons/warzone2100.desktop~	2008-11-27 10:06:44.000000000 +0100
+++ warzone2100-2.1_rc1/icons/warzone2100.desktop	2008-11-27 10:09:29.000000000 +0100
@@ -5,9 +5,13 @@
 Name[de]=Warzone 2100
 GenericName=Strategy Game
 GenericName[de]=Strategiespiel
+GenericName[pl]=Gra Strategiczna
 Comment=Postnuclear realtime strategy
 Comment[de]=Postnukleare Echtzeit Strategie
-Icon=warzone2100
+Comment[pl]=Gra strategiczna czasu rzeczywistego
+Icon=warzone2100.png
 TryExec=warzone2100
 Exec=warzone2100
 Categories=Game;StrategyGame;
+# vi: encoding=utf-8
+
