--- warzone2100-2.0.10/icons/warzone2100.desktop.orig	2007-12-04 21:05:25.000000000 +0100
+++ warzone2100-2.0.10/icons/warzone2100.desktop	2007-12-30 22:41:18.000000000 +0100
@@ -3,11 +3,15 @@
 Type=Application
 Name=Warzone 2100
 Name[de]=Warzone 2100
+GenericName[pl]=Gra Strategiczna
 GenericName=Strategy Game
 GenericName[de]=Strategiespiel
 Comment=Postnuclear realtime strategy
 Comment[de]=Postnukleare Echtzeit Strategie
+Comment[pl]=Gra strategiczna czasu rzeczywistego
 Icon=warzone2100
 TryExec=warzone2100
 Exec=warzone2100
 Categories=Game;StrategyGame;
+Categories=Game;StrategyGame;
+# vi: encoding=utf-8
